recipes/wan2.2-5b-kelpie/container_template.readme.mdx

```diff
@@ -0,0 +1,136 @@
+- `--id` (string, optional) — job id for logs/filenames; defaults to a random UUID.
+
+> The worker fixes `--task ti2v-5B`, always enables `--offload_model` and `--t5_cpu`, and writes to `/opt/outputs/<output-filename or job-id>.mp4`.
+
+
+## Quick Start — Text → Video
+
+Replace the placeholders and run:
+
+```bash
+export SALAD_API_KEY="<salad-api-key>"
+export SALAD_ORGANIZATION="<organization>"
+export SALAD_PROJECT="<project>"
+
+curl -s -X POST "https://kelpie.saladexamples.com/jobs" \
+  -H "Content-Type: application/json" \
+  -H "Salad-Api-Key: $SALAD_API_KEY" \
+  -H "Salad-Organization: $SALAD_ORGANIZATION" \
+  -H "Salad-Project: $SALAD_PROJECT" \
+  -d @- <<'JSON'
+{
+  "container_group_id": "<container_group_id>",
+  "command": "python",
+  "arguments": [
+    "/opt/wan_worker.py",
+    "--prompt", "A moody cyberpunk street in the rain, neon reflections, shallow depth of field",
+    "--size", "1280*704",
+    "--frame-num", "120",
+    "--sample-steps", "50"
+  ],
+  "sync": {
+    "after": [
+      { "bucket": "<bucket>", "prefix": "outputs/", "local_path": "/opt/outputs/", "direction": "upload" }
+    ]
+  }
+}
+JSON
+```
+
+## Quick Start — Text → Video
+
+Upload your reference image to s3 bucket
+
+Submit a job that downloads it into assets/ first:
+
+curl -s -X POST "https://kelpie.saladexamples.com/jobs" \
+  -H "Content-Type: application/json" \
+  -H "Salad-Api-Key: $SALAD_API_KEY" \
+  -H "Salad-Organization: $SALAD_ORGANIZATION" \
+  -H "Salad-Project: $SALAD_PROJECT" \
+  -d @- <<'JSON'
+{
+  "container_group_id": "<container_group_id>",
+  "command": "python",
+  "arguments": [
+    "/opt/wan_worker.py",
+    "--id", "job-002",
+    "--prompt", "Dreamy golden-hour dolly shot through wildflowers",
+    "--size", "1280*704",
+    "--frame-num", "120",
+    "--sample-steps", "50",
+    "--image-path", "assets/input.jpg"
+  ],
+  "sync": {
+    "before": [
+      { "bucket": "<bucket>", "prefix": "assets/", "local_path": "/opt/assets/", "direction": "download" }
+    ],
+    "after": [
+      { "bucket": "<bucket>", "prefix": "outputs/", "local_path": "/opt/outputs/", "direction": "upload" }
+    ]
+  }
+}
+JSON
+```
+## Monitor a job
+
+```bash
+curl -s "https://kelpie.saladexamples.com/jobs/<kelpie-job-id>" \
+  -H "Salad-Api-Key: $SALAD_API_KEY" \
+  -H "Salad-Organization: $SALAD_ORGANIZATION" \
+  -H "Salad-Project: $SALAD_PROJECT" | jq .
+```
+## Troubleshooting
+
+- **Container exits immediately** — Missing/invalid Salad credentials. Verify API key, organization, and project headers.
+- **Job “succeeded” but no MP4** — Check your output folder path.
+- **Image ignored** — Bad base64 or unreachable URL. Check if the image exist and correctly specified.
+- **No output uploaded** — ensure your job includes the `sync.after` rule for `outputs/`.
+
+> Need to scale automatically? Kelpie supports **queue-aware autoscaling**, including **scale-to-zero** when the queue is empty.
```

Comment:
We should mention that they need to add me to their org for autoscaling to work.